tasks/44_time-simulation/time-simulation.js

```diff
@@ -0,0 +1,18 @@
+export function SimTime() {
+  let simTime = 0;
+  let realTime = 0;
+  let speed = 1;
+
+  SimTime.prototype.get = function() {
+    return simTime;
+  };
+
+  SimTime.prototype.setSpeed = function(newSpeed) {
```

Comment:
`this. setSpeed = ...` will be enough

Reply:
I was requested to use prototypes by alexkond, though

Reply:
Yes, but it should be done outside of the constructor function.

Reply:
Ok